libduckdb-sys/src/bindgen_bundled_version.rs

```diff
@@ -1,3 +1,3562 @@
led on the result.\n\n database: The database object to create the task state for\n returns: The task state that can be used with duckdb_execute_tasks_state."] pub fn duckdb_create_task_state (database : duckdb_database) -> duckdb_task_state ; } extern "C" { # [doc = "Execute DuckDB tasks on this thread.\n\nThe thread will keep on executing tasks forever, until duckdb_finish_execution is called on the state.\nMultiple threads can share the same duckdb_task_state.\n\n state: The task state of the executor"] pub fn duckdb_execute_tasks_state (state : duckdb_task_state) ; } extern "C" { # [doc = "Execute DuckDB tasks on this thread.\n\nThe thread will keep on executing tasks until either duckdb_finish_execution is called on the state,\nmax_tasks tasks have been executed or there are no more tasks to be executed.\n\nMultiple threads can share the same duckdb_task_state.\n\n state: The task state of the executor\n max_tasks: The maximum amount of tasks to execute\n returns: The amount of tasks that have actually been executed"] pub fn duckdb_execute_n_tasks_state (state : duckdb_task_state , max_tasks : idx_t) -> idx_t ; } extern "C" { # [doc = "Finish execution on a specific task.\n\n state: The task state to finish execution"] pub fn duckdb_finish_execution (state : duckdb_task_state) ; } extern "C" { # [doc = "Check if the provided duckdb_task_state has finished execution\n\n state: The task state to inspect\n returns: Whether or not duckdb_finish_execution has been called on the task state"] pub fn duckdb_task_state_is_finished (state : duckdb_task_state) -> bool ; } extern "C" { # [doc = "Destroys the task state returned from duckdb_create_task_state.\n\nNote that this should not be called while there is an active duckdb_execute_tasks_state running\non the task state.\n\n state: The task state to clean up"] pub fn duckdb_destroy_task_state (state : duckdb_task_state) ; } extern "C" { # [doc = "Returns true if the execution of the current query is finished.\n\n con: The connection on which to check"] pub fn duckdb_execution_is_finished (con : duckdb_connection) -> bool ; } extern "C" { # [doc = "Fetches a data chunk from the (streaming) duckdb_result. This function should be called repeatedly until the result is\nexhausted.\n\nThe result must be destroyed with `duckdb_destroy_data_chunk`.\n\nThis function can only be used on duckdb_results created with 'duckdb_pending_prepared_streaming'\n\nIf this function is used, none of the other result functions can be used and vice versa (i.e. this function cannot be\nmixed with the legacy result functions or the materialized result functions).\n\nIt is not known beforehand how many chunks will be returned by this result.\n\n result: The result object to fetch the data chunk from.\n returns: The resulting data chunk. Returns `NULL` if the result has an error."] pub fn duckdb_stream_fetch_chunk (result : duckdb_result) -> duckdb_data_chunk ; } pub type __builtin_va_list = * mut :: std :: os :: raw :: c_char ;
```

Comment:
not sure why the old bindings had everything generated on one line - feel free to run your own `upgrade.sh` and push to this branch if that's the desired behavior

Reply:
In any case, I much prefer the new formatted version for several reasons. One of them being that go-to source will be actually helpful now. 😄